File: nodejs/8 kyu/152.js
// https://www.codewars.com/kata/570597e258b58f6edc00230d/train/javascript

// Passed

function array(string) {
  let commaSeparatedCharacters = string.split(",");
  if (
    commaSeparatedCharacters.length === 0 ||
    commaSeparatedCharacters.length === 1 ||
    commaSeparatedCharacters.length === 2
  )
    return null;

  let result = commaSeparatedCharacters
    .slice(1, commaSeparatedCharacters.length - 1)
    .join(" ");
  return result;
}

const output = array("1,2,3,4");
console.log(output);

File: python/7 kyu/152.py
# https://www.codewars.com/kata/595877be60d17855980013d3/train/python

# Passed

def euclidean_distance(point1, point2):
    dist = 0
    for coordinate1, coordinate2 in zip(point1, point2):
        dist += (coordinate2 - coordinate1) ** 2

    dist = round(dist ** 0.5, 2) # taking the square root
    return dist

output = euclidean_distance([-1, 2, 5], [2, 4, 9])
print(output)